src/Panel/Page.php

```diff
@@ -120,16 +120,14 @@ public function dropdown(array $options = []): array
             'disabled' => $this->isDisabledDropdownOption('changeStatus', $options, $permissions)
         ];
 
-        if ($view === 'list') {
```

Comment:
You want to show this option now as well on the page itself, not just in list view?

Reply:
Yesterday when I tested it, I tried it in list view and then switched to the page view and wondered why the option isn't there at all. It felt like a bug. In this case the status dialog was disabled via permissions but sorting was still possible. So there was no way to change the position of the page via the page view at all.

Reply:
👍